x/mongo/driver/errors.go

```diff
@@ -91,6 +92,11 @@ func (wce WriteCommandError) Error() string {
 
 // Retryable returns true if the error is retryable
 func (wce WriteCommandError) Retryable() bool {
+	for _, label := range wce.Labels {
```

Comment:
Can you explain this change? On a network error, the wire message sending code always returns an `Error` with the `NetworkError` label. If we got a `WriteCommandError`, that already means the network communication was successful, so we should never hit this case.

Reply:
I was pretty sure I didn't need that check, but I wasn't sure so I added it in case. Removed.